typescript/index.ts

```diff
@@ -0,0 +1,48 @@
+export interface Workflow {
```

Comment:
`typescript` doesn't feel like the right directory name here. a casual reader of the workflows repo may not know what this folder is for. alternative names: `js_export` or `typescript_export` or `npm_package`

Reply:
agreed here--`build_ts` sgtm :)